packages/packem/src/validator/validate-bundle-size.ts

```
@@ -0,0 +1,95 @@
+
+            // eslint-disable-next-line no-continue
+            continue;
+        }
+
+        if (typeof limit === "string") {
+            limit = parseBytes(limit);
+        }
+
+        if (!Number.isFinite(limit) || limit <= 0 || (foundEntry.size.bytes as number) <= limit) {
+            // eslint-disable-next-line no-continue
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)